### DISCUSSION TITLE
Approve Actions is not working after upgrade to 7.8.2.1

### DISCUSSION THREAD

**Discussion by zmmm85** [Keyfactor/ejbca-ce#34] [Q&A] [answered]
Hello, I have upgraded EJBCA from 6.5 to 7.8.2.1, but the Approve Action tab is not working. There is nothing in the log indicate any errors. 
Any ideas or suggestions related to this issue?

Note: the issue didn't happen when I upgrade to 7.4 ce

Many Thanks,

**Comment by mike-agrenius-kushner** [comment, marked as answer]
Hi zmmm. May I call you Z? 

We've been trying to separate our CA and RA interfaces for a while now (to avoid as much redundancy as possible), so a few versions ago we moved all enrollment-related approval activities to the RA UI. Go in there and check under the Approvals tab and I think you'll find them there. 

Cheers,
Mike

**Comment by zmmm85** [comment]
Thank you so much Mike for your quick answer and sure you can call me Z.
Yes, I can see all certificates under RA functions.
Just have one more questions, why at the home tab, the version says Enterprise while I downloaded ejbca ce?

Thanks again,
Z
